manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.1.0 <4.2.0"
+    olm.skipRange: ">=4.2.0 <4.3.0"
```

Comment:
same question here as EO... should we go from `>=4.1.0 <4.3.0`?

Reply:
No.  This is intentional and fits our policy with prior releases to force stepwise upgrades